odoo/addons/base/models/res_partner.py

```diff
@@ -196,11 +196,11 @@ def default_get(self, default_fields):
     country_id = fields.Many2one('res.country', string='Country', ondelete='restrict')
     partner_latitude = fields.Float(string='Geo Latitude', digits=(16, 5))
     partner_longitude = fields.Float(string='Geo Longitude', digits=(16, 5))
-    email = fields.Char()
+    email = fields.Char(string='Email', tracking=True)
```

Comment:
Tracking parameter makes sense only in mail. It makes no sense to use it in base. You will have to override the fied in mail to add the parameter. Note that you can also give an integer to order tracking values. Could be great to use it.